.eslintrc.js

```diff
@@ -1,15 +1,16 @@
 /** @type {import('@types/eslint').Linter.BaseConfig} */
 module.exports = {
   extends: [
-    "@remix-run/eslint-config",
-    "@remix-run/eslint-config/node",
-    "@remix-run/eslint-config/jest-testing-library",
-    "prettier",
+    'eslint:recommended',
+    '@remix-run/eslint-config',
+    '@remix-run/eslint-config/node',
+    '@remix-run/eslint-config/jest-testing-library',
+    'prettier',
   ],
   env: {
-    "cypress/globals": true,
+    'cypress/globals': true,
   },
-  plugins: ["cypress"],
+  plugins: ['cypress'],
```

Comment:
same thing regarding the `cypress` is there a good reason to have it here?

Reply:
I wasn't sure if I should remove it considering someone was already given a task regarding removal of cypress and tailwind from our repo

Reply:
I would suggest to remove those too

Reply:
same as above